### PR TITLE
Joren/fix/28 unhandled chrash on member creation

### PR DESCRIPTION
**Pull request by Joren-Thijs**
Fixed input issue by constraining range of number input.

### CODE REVIEW

**Comment by Joren-Thijs** [comment]
I'm working in the browser again so i can't test it yet. I will test it when i get home👍

File: Web/Pages/Members/MemberForm.razor [hunk 1]
```diff
@@ -37,8 +37,8 @@
         </MudDatePicker>
     </MudItem>
     <MudItem xs="12" sm="6">
-        <MudNumericField Label="Monthly membership fee" @bind-Value="Member.MembershipFee" Format="F2"
-            Adornment="Adornment.Start" AdornmentIcon="@Icons.Material.Filled.EuroSymbol"
+        <MudNumericField Label="Monthly membership fee" @bind-Value="Member.MembershipFee" Min="0" Max="1000"
```
Comment (vyruz1986) [inline Web/Pages/Members/MemberForm.razor:40]: Can't this validation be done on the MemberFrom class? That is where all other validation is located

**Review by vyruz1986 — CHANGES_REQUESTED** [review]

File: Web/Pages/Members/MemberForm.razor (hunk 1 above)
Comment (Joren-Thijs) [reply Web/Pages/Members/MemberForm.razor:40]: It already has a range attribute added 3 months ago but it does not seem to be picked up by the MudBlazor component

Comment (Joren-Thijs) [reply Web/Pages/Members/MemberForm.razor]: https://github.com/Brixel/HaSpMan/blob/86a8fb1c5450f5f78e05f3aeff30aac3f6c46a6f/Web/Models/MemberForm.cs#L47

**Comment by Joren-Thijs** [comment]
I tested it locally and the input field now behaves correctly. Changing a value of -5 to 0.00 and a value of 2000 to 500.00👍

File: Web/Pages/Members/MemberForm.razor (hunk 1 above)
Comment (vyruz1986) [reply Web/Pages/Members/MemberForm.razor]: In that case you should research why the data annotation validation isn't working. We validate all other input fields using data annotations, so either it's a bug in MudBlazor, or we're doing something wrong

Comment (Joren-Thijs) [reply Web/Pages/Members/MemberForm.razor]: Iss withing to fluent validation an option? the documentation clearly states it's possible

Comment (vyruz1986) [reply Web/Pages/Members/MemberForm.razor]: Let's wait with introducing FluentValidations until we really have a need for them.
I did a quick test, starting from MudBlazors EditForm input validation example, and copy/pasted our MembershipFee field into the example code, and that works fine: https://try.mudblazor.com/snippet/QkclbQcCVrTkdLAd

**Review by vyruz1986 — APPROVED** [review]

**Pull request merged**